Trippy/View Controllers/Itinerary Tab/PreferencesViewController.m

```diff
@@ -82,12 +82,13 @@ - (IBAction)tapView:(id)sender {
 - (IBAction)tapUpdate:(id)sender {
     NSDate *etaStart = self.prefEtaSwitch.isOn ? self.etaStartPicker.date : [NSNull null];
     NSDate *etaEnd = self.prefEtaSwitch.isOn ? self.etaEndPicker.date : [NSNull null];
-    NSNumber *stayTime = self.estStaySwitch ? [self stayTimeToSeconds] : @0;
+    TimeInHrMin fieldTime = {.hours = [self.stayHrField.text intValue], .minutes=[self.stayMinField.text intValue]};
```

Comment:
Not a big fan of getting values from text fields... can you pull from the source instead? You might even already have it in seconds.

Reply:
I'm getting an input value from the user, so I'm not sure what you mean by source?

Reply:
AH! Ok, no worries then 😄 disregard!